components/camel-pubnub/pom.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Licensed to the Apache Software Foundation (ASF) under one or more contributor 
```

Comment:
Add nicer license header, you can copy from one of the other existing pom.xml files